### PR TITLE
Milestone 3 react calculator app

### PR DESCRIPTION
**Pull request by kalavhan**
Pull request for the review of the third milestone of the Calculator react app
Changes added to this pull request:

- Width of the calculator set to 700px
- height of display and rows of the button set to 100px
- Add wide and color props to the button component
- color by default orange for buttons
- 25% of width by default to the buttons

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+  margin-top: 10px;
+}
+
+.calculator {
+  width: 700px;
+  display: flex;
+  flex-direction: column;
+}
+
+.display {
```
Comment (Oghenebrume50) [inline src/index.css:14]: This should use flex display 
see

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @kalavhan good job on implementing the design :slightly_smiling_face: everything looks good except a small caveat to make this pass, I have left a comment below. 
**please make changes and request for another review**  :+1: 

**RAPHAEL NORIODE** :mask:

**Comment by kalavhan** [comment]
@Oghenebrume50 Thanks a lot for the review, I pushed the requested changes:
.calculator is the class for the app
`.calculator {
  width: 700px;
  display: flex;
  flex-direction: column;
}`

.display was missing the display:flex property, now added
`.display {
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  text-align: right;
  background-color: #808080;
  height: 100px;
  color: #fff;
  font-weight: 900;
  font-size: 80px;
}`

.buttonPanel was missing the display:flex property, now added
`.buttonPanel {
  display: flex;
  flex-direction: column;
}`

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        wide
+      />
+      <Button
+        name="."
+        color="light-gray"
+      />
+      <Button
+        name="="
+      />
+    </div>
```
Comment (shshamim63) [inline src/components/ButtonPanel.js:87]: [Suggestion]
This block of code is long and contains a lot of repetition. You can refactor it. How? Well, I can give you a hint. Think about an array contains arrays that contain a group of buttons symbol. Then you can iterate it the array.

**Review by shshamim63 — APPROVED** [review]
## :100: Approved :100: 
___
Hi @kalavhan :wave: ,
Congratulation, you have covered all the tasks asked by the previous reviewer. Just adding a suggestion to make your code dry, check out the comment below. In summary, Nothing is blocking me to approve this milestone now. This :+1: is from me. You can go ahead.
___
## :herb: Happy Coding :herb: 
### Shakhaway Hossain

**Pull request closed without merging**